systemtests/bin/upload_symbols.py

```diff
@@ -15,7 +15,8 @@
 import click
 import markus
 from markus.backends import BackendBase
-import requests
+
```

Comment:
Since there's already a retry loop, I wanted to just stick with that rather than think through the specific situations that are retry-friendly for HTTP verbs that can be destructive. We should definitely tackle a retry-friendly POST. I'm not sure if we could write a generic one we could use most places or if we have to make each one personalized.

I can think through that for this PR, though. You're right in that it's weird I didn't do a retryable post.